docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -345,6 +346,66 @@ If the script finishes successfully, go to the location you provided with the
 > location as an existing file. To allow overwriting, add the `--overwrite`
 > option to the argument list of the Python script.
 
+### How to set user data
+
+The user data is an optional field provided in factory data JSON file and depends on user's or manufacturer's purpose (or both). Currently, we support two data types that can be placed in user data field: `string` and `int32`.
```

Comment:
We seem to lack the description that user is a JSON map whose values are restricted to strings and int32. Without that info a user might think that `user` itself should be string or int32.